### DISCUSSION TITLE
Existing Library

### DISCUSSION THREAD

**Discussion by nikhilhurule** [immich-app/immich#1157] [General]
Hey new here is there any way to import existing library to immich.

**Comment by alextran1502** [comment]
You can find the CLI tool for bulk import in the documentation 

https://immich.app/docs/features/bulk-upload

**Reply by nikhilhurule** [reply]
does the import tool  just imports them or copy them (i have a very big lib. about 15K+ photos ) to a diffrent location because i have limited storage.

**Reply by alextran1502** [reply]
They will be uploaded to Immich, so make a copy. You will have to make some space for that.

**Reply by nikhilhurule** [reply]
Thanks man great work really love the project
